documents-webapp/src/main/webapp/vue-app/documents-extensions/components/DocumentsFavoriteItem.vue

```diff
@@ -46,6 +46,7 @@ export default {
   created() {
     this.$attachmentService.getAttachmentById(this.id)
       .then(file => { 
+        file.title = decodeURI(file.title); 
         this.file = file;
         this.documentTitle = decodeURI(file.title);
```

Comment:
there is already a decoding of property documentTitle which takes the value of file.title. which one is used in our case ? do we need both of properties ?

Reply:
One call of the decodeURI method won't solve the problem. That's why I encode the title and the document title. I changed this solution with a double call of decodeURI on the documentTitle.